utils/featureFlags.ts

```diff
@@ -0,0 +1,28 @@
+import * as t from "io-ts";
+
+export const FeatureFlag = t.union([
+  t.literal("all"),
+  t.literal("beta"),
+  t.literal("canary"),
+  t.literal("none")
+]);
+export type FeatureFlag = t.TypeOf<typeof FeatureFlag>;
```

Comment:
Please, move to `enumType` like [here](https://github.com/pagopa/io-ts-commons/blob/3ec0e9bdf706ac22d02853c390e8317392299617/src/environment.ts#L12).
This way we can avoid fixed string and use enum value instead.